common/scripts/generate-doc.js

```diff
@@ -36,11 +36,64 @@ function walk(dir, checks) {
   return checks;
 }
 
+function buildList(inputString) {
+  var inputList = inputString[1].split(",");
+  //console.log("exclusion list:" + exclusionList);
+  for (var i in inputList) {
```

Comment:
This function is super offputting to me, can you reply with what an input would look like to this?

Reply:
Removed this function in favour of arguments handler package for js => `yargs`